docs/omero_figure_move.rst

```diff
@@ -0,0 +1,99 @@
+Move figures between groups
+===========================
+
+Moving figures between groups in OMERO might be necessary for example in cases where a scientist decides to move all their data into another group to enable cooperation and viewing by other colleagues or to a group which is to be made public later. This walkthrough offers two alternatives, with or without duplicating the figures and the images contained in them.
+
+Description
+-----------
+
+This guide covers:
+
+- Manual moving of figures using user interface.
+- Retrieval of all the images contained in a Figure.
+- Duplicating of images contained in a Figure, moving them and re-creating Figure duplicates in the target group. 
```

Comment:
Figure(s) and Images should all start with upper-case throughout the guide.

Reply:
Same for "Group"

Reply:
As suggested below, I think it's clearer to divide the guide into only 2 walkthroughs, named simply to emphasize the Move/duplicate difference between them:
- Moving of Figures and Images between Groups
- Duplicating of Figures and Images into a new Group

Reply:
fixed in https://github.com/ome/omero-guide-figure/pull/13/commits/4eb8f01b2bd0f64dcb0ae49158bc6592b117988c